dao/dao_base.go

```diff
@@ -34,6 +34,7 @@ type DaoWrapper struct {
 	// AuditDao.Find(...) seems like a nice api, find can be used in other dao as well if type is not embedded
 	AuditDao AuditDao
 	InfoPageDao
+	BookmarkDao
```

Comment:
So we can use function names like `Add` in the other daos as well:

```suggestion
	BookmarkDao BookmarkDao
```